pkg/controller/ingress-shim/sync.go

```diff
@@ -169,6 +176,12 @@ func certNeedsUpdate(a, b *v1alpha1.Certificate) bool {
 		return true
 	}
 
+	if a.Spec.ACME != nil && b.Spec.ACME != nil {
```

Comment:
I don’t think this brace is required. What if the Certificate is changing from a non ACME cert to an ACME one?

Reply:
it was to protect against nil pointer errors more than anything else, I've replaced it with individual checks